features/step_definitions/course_listing_steps.rb

```diff
@@ -0,0 +1,59 @@
+require 'watir-webdriver'
+
+#Scenario: Searching for CSCI 2994
+Given /^I am on the CSCC homepage$/ do
+  #Open browser and goto the CSCC homepage
+  @browser = Watir::Browser.new
+  @browser.goto "www.cscc.edu"
+end
+
+When /^I search for csci 2994$/ do
+  @browser.text_field(:id => "input").set "csci 2994"
+  @browser.button(:id => "submit").click

```

Comment:
You should be using a page object instead of calling watir-webdriver methods directly.